testsuite/MDAnalysisTests/core/test_atomselections.py

```diff
@@ -1218,3 +1219,60 @@ def test_record_type_sel():
 
     assert len(u.select_atoms('name CA and not record_type HETATM')) == 30
     assert len(u.select_atoms('name CA and record_type HETATM')) == 2
+
+
+def test_element_sel():
+    # test auto-topattr addition of string
+    u = mda.Universe(PDB_elements)
+    assert len(u.select_atoms("element H")) == 8
+    assert len(u.select_atoms("same element as index 12")) == 8
+
+
+def test_chain_sel():
+    u = mda.Universe(PDB_elements)
+    assert len(u.select_atoms("chainID A")) == len(u.atoms)
+
+
+@pytest.mark.parametrize("selstr,n_atoms", [
+    ("mass 0.8 to 1.2", 23850),
+    ("mass 8e-1 to 1200e-3", 23850),
+    ("mass -5--3", 2),  # select -5 to -3
+    ("mass -3 : -5", 0),  # wrong way around
+])
+def test_mass_sel(selstr, n_atoms):
+    # test auto-topattr addition of float (FloatRangeSelection)
```

Comment:
I think for floats some tests that rounding errors aren't breaking equality checks would be good

Reply:
Or that they do as one might expect ;-)